main.go

```diff
@@ -10,8 +10,8 @@ import (
 	"net/url"
 	"os"
 
-	"github.com/fiorix/wsdl2go/wsdl"
-	"github.com/fiorix/wsdl2go/wsdlgo"
+	"github.com/Kernle32DLL/wsdl2go/wsdl"
```

Comment:
Bad imports will cause compilation errors.

When I fork go code I normally keep the original package path and just add `origin` to the local git config pointing to my fork. And then add `upstream` pointing to the fork's origin.